Backend/TimeTracker.Api/Controllers/ProfileController.cs

```diff
@@ -0,0 +1,111 @@
+using Microsoft.AspNetCore.Mvc;           // ControllerBase
```

Comment:
You can probably remove all the comments here. Not sure if you have an extension that automatically adds them.